src/System.IO.FileSystem/tests/Performance/System.IO.FileSystem.Performance.Tests.csproj

```diff
@@ -26,6 +31,9 @@
     <Compile Include="$(CommonTestPath)\System\IO\PathFeatures.cs">
       <Link>Common\System\IO\PathFeatures.cs</Link>
     </Compile>
+    <Compile Include="$(CommonTestPath)\System\PlatformDetection.cs">
```

Comment:
@hughbe any test common code should be moved to the TestUtilities library. are you interested doing that? that will be fantastic.